polygon/p2p/message_listener.go

```diff
@@ -0,0 +1,141 @@
+package p2p
+
+import (
+	"context"
+	"sync"
+
+	"github.com/ledgerwatch/log/v3"
+	"google.golang.org/grpc"
+
+	"github.com/ledgerwatch/erigon-lib/direct"
+	"github.com/ledgerwatch/erigon-lib/gointerfaces/sentry"
+	sentrymulticlient "github.com/ledgerwatch/erigon/p2p/sentry/sentry_multi_client"
+)
+
+type MessageListener interface {
+	Start(ctx context.Context)
+	Stop()
+	RegisterBlockHeaders66Observer(observer MessageObserver[*sentry.InboundMessage])
+	UnregisterBlockHeaders66Observer(observer MessageObserver[*sentry.InboundMessage])
```

Comment:
```suggestion
	RegisterBlockHeadersObserver(observer MessageObserver[*sentry.InboundMessage])
	UnregisterBlockHeadersObserver(observer MessageObserver[*sentry.InboundMessage])
```

Suggest to remove 66 (eth version) from method names here, because it is a lower-level concern.

Reply:
addressed - https://github.com/ledgerwatch/erigon/pull/9470